lib/server.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+const express = require('express');
+const app = express();
+const timestamp = require('../middleware/timestamp');
+const logger = require('../middleware/logger');
```

Comment:
There's something else you have to do with this middleware you created, in order for them to work properly.

Hint: `app.use(express.json())` is bringing in middleware